handbook/digital-experience/digital-experience.rituals.yml

```
@@ -17,6 +17,13 @@
   description: "Prepare the CEO office minutes calendar event and meeting agenda"
   moreInfoUrl: "https://fleetdm.com/handbook/digital-experience#prepare-for-ceo-office-minutes"
   dri: "sampfluger88"
+- 
+  task: "Refresh event calendar"
+  startedOn: "2023-12-31"
+  frequency: "Quarterly"
+  description: "At the end of every quarter, the PR firm updates every event in the Event strategy workbook
```

your way is nice, I'm just leaning towards being lazy
(also if we capitalize "E", then use double-quotes.)

But really, I think we just go the lazy way for now?
https://fleetdm.com/handbook/digital-experience#refresh-event-calendar

```suggestion
  description: "https://fleetdm.com/handbook/digital-experience#refresh-event-calendar"
```

---

I'm thinking at current resource levels, the advantages of deduping outweigh the inconvenience of having to follow the link to see what the ritual is all about